lib/chef/knife/bootstrap.rb

```diff
@@ -296,6 +296,12 @@ class Bootstrap < Knife
         description: "URL to a custom installation script.",
         proc: Proc.new { |u| Chef::Config[:knife][:bootstrap_url] = u }
 
+      option :bootstrap_project,
+        long: "--bootstrap-project PROJECT",
+        description: "Project to install.",
+        proc: Proc.new { |u| Chef::Config[:knife][:bootstrap_project] = u },
```

Comment:
you shouldn't need to do this Proc nonsense, even though we haven't ripped it out everywhere yet.

Reply:
I see. removed.